tutorials/fundamentals/remove-last-char.md

```diff
@@ -0,0 +1,33 @@
+To remove the last charcter from a string in JavaScript, you should use the `slice()` method.
+It takes two arguments, what index to start and what index to end.
+`slice()` has negative indexing, which means that you could do `slice(0, -1)` and it would be equivalent to `slice(0, str.length - 1)`.
```

Comment:
"you could do X and it would be equivalent to Y" -> "X, which is equivalent to Y"